Makefile

```diff
@@ -113,24 +113,20 @@ clang-tidy-ci:
 	$(call mkdirp,build/release) && cd build/release && \
 	cmake $(GENERATOR) $(FORCE_COLOR) $(SANITIZER_FLAG) -DCMAKE_EXPORT_COMPILE_COMMANDS=1 ../..
 
-pytest:
-	$(MAKE) release
+pytest: release
```

Comment:
If we make this change, will it still rerun `make release` every time  when`make pytest` is run? Since `release` does not have file dependencies configured and is simply a shortcut to invoke CMake, I am not sure if it will rebuild automatically if the code is changed?

Reply:
release is a dummy target, so Make will run it everytime pytest is run